src/pocketmine/block/SeaLantern.php

```diff
@@ -49,4 +49,4 @@ public function getDrops(Item $item){
 		];
 	}
 
-}
```

Comment:
:facepalm: i meant the newline at the end of the file. It didn't contribute to the fix did it?